android/src/main/java/com/flutter_webview_plugin/WebviewManager.java

```diff
@@ -495,4 +498,13 @@ void stopLoading(MethodCall call, MethodChannel.Result result){
             webView.stopLoading();
         }
     }
+
+    public class WebAppInterface {
+        @JavascriptInterface
+        public void postMessage(String value){
+            Map<String, String> postMessageMap = new HashMap<>();
+            postMessageMap.put("postMessage", value);
+            FlutterWebviewPlugin.channel.invokeMethod("onPostMessage", postMessageMap);
```

Comment:
I have a error:
java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: JavaBridge
maybe use 'new Handler(Looper.getMainLooper())'

Reply:
done

Reply:
How did you solve this error 
?
I have a error:
java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: JavaBridge
maybe use 'new Handler(Looper.getMainLooper())'
I am getting the same

Reply:
I add mainLooper in [commit](https://github.com/fluttercommunity/flutter_webview_plugin/pull/457/commits/62872ced09f8dce9119f5119086437090f01984e#diff-2965422868f060f130659de350f7c684R510), are you sure on the lastest branch?

Reply:
you can look #523.

Reply:
#523 is already implemented on the main branch